interface-definitions/service_monitoring_telegraf.xml.in

```diff
@@ -85,6 +85,89 @@
                 </properties>
                 <defaultValue>all</defaultValue>
               </leafNode>
+              <node name="prometheus-client">
+                 <properties>
+                   <help>Output plugin Prometheus client</help>
+                 </properties>
+                 <children>
+                   <node name="authentication">
+                     <properties>
+                       <help>HTTP basic authentication parameters</help>
+                     </properties>
+                     <children>
+                       <leafNode name="username">
+                         <properties>
+                           <help>Authentication username</help>
```

Comment:
No regex required on username? We should not allow UMLAUT characters and stick with the regular linux user regex: `<regex>[-_a-zA-Z0-9.]{1,100}</regex>` Maybe username can be moved to an include if it's re-used more frequently?

Reply:
Done!